SharePointFramework/ProjectWebParts/src/components/ProjectPhases/getPhaseSitePages.ts

```diff
@@ -0,0 +1,30 @@
+import { ProjectPhaseModel } from 'pp365-shared/lib/models'
+import { sp } from "@pnp/sp";
+import { IPhaseSitePageModel } from './types';
+
+/**
+ * Change phase
+ *
+ * @param {ProjectPhaseModel} phases
+ */
+export const getPhaseSitePages = async (
+  phases: ProjectPhaseModel[]
+) => {
+  try {
+    let sitepages = await sp.web.lists.getByTitle('Områdesider').items.select("Id, Title, FileRef, EncodedAbsUrl, FileLeafRef").get()
```

Comment:
Consistent naming. We're using `phaseSitePages`, so it should be `sitePages`.

Reply:
Can you elaborate this?

Reply:
Nvm, saw the camelcase now

Reply:
We should use consistent variable naming.

`phaseSitePages` => camelCase
`sitepages` => lowercase